brave-lists/debounce.json

```diff
@@ -25,7 +25,8 @@
       "*://go.redirectingat.com/*",
       "*://x.chip.de/linktrack/button/*",
       "*://*.mailchimp.com/mctx/clicks?*",
-      "*://*.tradedoubler.com/click?*"
+      "*://*.tradedoubler.com/click?*",
+      "*://cc.techbargains.com/*"
```

Comment:
I would include this part of the path as well: `/v1/otc/`, in other words:
```
      "*://cc.techbargains.com/v1/otc/*"
```
If they change to another version of their API (e.g. `/v2/otc/...`) then we may need to revise the rule type or the name of the parameter.